### DISCUSSION TITLE
MinIO Caching

### DISCUSSION THREAD

**Discussion by AmirMohG** [minio/minio#15799] [Q&A] [answered]
Hi everyone.
I've recently started working on a MinIO deployment task (4 nodes 4 drives test environment). I don't seem to understand the reason behind the caching feature which caches objects in a directory on the servers. The only example I see in the docs is with gateways which seem to be deprecated. I'd greatly appreciate it if someone could refer to a link or write an explanation. Thanks in advance.

**Comment by harshavardhana** [comment, marked as answer]
Caching is only meant for gateways not distributed server deployment.

**Reply by AmirMohG** [reply]
Thanks for the confirmation.
